functions/src/index.ts

```diff
@@ -25,5 +25,8 @@ try {
 import * as SQR from './modules/SQR.functions';
 import * as ReportingSync from './modules/reporting.sync.functions';
 import * as SE from './modules/SE.functions';
+import * as Authorization from "./modules/Authorization.functions";
 
-export { SQR, ReportingSync, SE };
\ No newline at end of file
+export { SQR, ReportingSync, Authorization };
+
+export { SE }
```

Comment:
Are you sure a separate export is okay here?

Reply:
was just separating them until `Email` is merged into **Master**